docs/developer-docs/integrations/sns/managing/making-proposals.md

```diff
@@ -742,6 +789,20 @@ quill send message.json
 ```
 
 
+## Critical proposals
+
+Some proposal types are considered "critical". These are DeregisterDappCanisters, TransferSnsTreasuryFunds, and MintSnsTokens. Critical proposal types are handled differently to ensure they are only passed with broad community consensus.
+
+1. Voting Thresholds
+
+   Non-critical proposals types can be passed if 3% of the total voting power votes yes and 50% of the exercised voting power votes yes.
+
+   Critical proposals types can only be passed if 20% of the total voting power votes yes and 67% of the exercised voting power votes yes. 
+
+2. Catchall following
+
+   Users must set following explicitly for critical proposal types.
+
```

Comment:
One point is missing here: the vote period for critical proposals is also fixed an usually longer than for normal proposals.

Reply:
Ah, I knew I was forgetting something!